releasenotes/notes/fix-flask-2.3.0-9818330e39b287c2.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Remove Flask hooks deprecated in version 2.3.0
```

Should we make this a bit more descriptive and specific for customers? Something like
```suggestion
    flask: Remove patching for Flask hooks ``app.before_first_request`` and ``bp.before_app_first_request`` if Flask version >= 2.3.0.
```